src/compas_occ/geometry/curves/nurbs.py

```diff
@@ -7,6 +7,7 @@
 from compas.geometry import Frame
 from compas.geometry import Circle
 from compas.geometry import Box
+from compas.geometry._core.distance import distance_point_point
```

Comment:
`from compas.geometry import distance_point_point` (i assume this was an autoimport)

Reply:
yes....